zerobuf/Zerobuf.h

```diff
@@ -10,6 +10,7 @@
 #include <zerobuf/api.h>
 #include <zerobuf/Types.h>
 #include <servus/uint128_t.h>
+#include <zerobuf/jsoncpp/json/json.h>
```

Comment:
json to string to json does not seem proper to me. All to JSON tree than a single convert to string seems more covenient.